_posts/2019-05-20-JDK-distributions.md

```diff
@@ -0,0 +1,211 @@
+If you want pick a vendor, make sure you research how long they will be providing free updates/support because this may vary from vendor to vendor.
+Further in the article we provide a table with most of this information.
+
+| Version       | Release date  | Free updates ended or superseded                                                             |
+| ------------- | ------------- | -------------------------------------------------------------------------------------------- |
+| OpenJDK 6     | December 2006 | Supported primarily by Azul systems                                                          |
+| OpenJDK 7     | July 2011     | Supported primarily by Red Hat until at least June 2020                                      |
+| OpenJDK 8     | March 2014    | Supported by Red Hat, Amazon, Azul Systems, BellSoft, Google, IBM, jClarity, SAP, and others.|
+| OpenJDK 9     | Sept 2017     | Superseded by OpenJDK 10                                                                     |
+| OpenJDK 10    | March 2018    | Superseded by OpenJDK 11                                                                     |
+| OpenJDK 11    | Sept 2018     | Supported by Red hat, Amazon, Azul Systems, BellSoft, Google, IBM, jClarity, SAP, and others.|
+| OpenJDK 12    | March 2019    | Superseded by OpenJDK 13                                                                     |
+| OpenJDK 13    | Sept 2019     | Will be superseded by OpenJDK 14                                                             |
+
+
+### Support and licenses
+
+Before Java 9, Oracle would provide updates for their JDK for three years.
+The license allowed you to use these updates personally and commercially.
+The updates had no support so if you wanted support you had to buy a license from Oracle and those licenses also came with longer update cycles.
+Starting April 2019, the new Oracle JDK 8 updates will be restricted and will remain free for personal desktop use, development, testing, prototyping, demonstrating and for use with certain types of applications.
+Commercial use of these updates requires you to get a license for a paid support plan.
+Using older versions is allowed, but will not contain important security patches.
+
+Java 9 brings new update cycles which allows for new implementations like TLS 1.3 to be added to Java faster.
+Oracle also started producing Oracle OpenJDK builds which use the GNU General Public License, version 2, with the Classpath Exception.
+Every major Java version has free updates for six months until the next major version is released.
+If you need new updates and support for these older versions, you will have to buy a license from Oracle which will extend support and updates although they only provide this for LTS releases.
+
+#### Support
+
+So what if I want support from Oracle?
+Support isn't free commitment to fix bugs and requiring to answer users their problems costs money.
```

Comment:
This is a really weird sentence IMO. What are you trying to say?

Reply:
It's trying to explain that support isn't free because Oracle has to hire employees and developers to resolve tickets and fix bugs. We can probably describe this another way.

Reply:
```suggestion
Support isn't free Oracle and other companies have to hire employees and developers to provide these support services.
```

Reply:
That's better! You can then remove the next line too.
I'd add "because" after "Support isn't free" though, but that's up to you. :)